prerelease_script.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+
+def compilemessages(data):
+    try:
+        if data['workingdir'] != data['reporoot']:
+            raise EnvironmentError('Working dir is not the root one')
+        os.chdir("mds")
+        os.system("../manage.py compilemessages")
+    except EnvironmentError as error:
+        print('Error occured while compilemessages')
+        print('Working Dir: ' + data['workingdir'])
+        print(error)
```

Comment:
Are you sure ? I actually did that so that we could release because I would consider translations problems as minor things :'s

Reply:
Printing errors and continue the process is equivalent to ignoring it.
That's the most painful thing you can do to create nasty bugs.
Also `compilemessages` is really not failing often. If it does, it may actually point to an actual error.
If you do want to have a release without translations, you can and you **must** have to tweak the release process (because why would you want to do it ???)
So it's a big **yes I am**

Reply:
I'm 200% sure we want the actual traceback and we don't want a special release without the translations.